### PR TITLE
contents: add Cover Letter section to resume page

### PR DESCRIPTION
**Pull request by figo-rabo**
#### Description:
The branch `feature/add-section-CV-cover-letter` adds 900 words to the `resume.md`. The changes provide valuable advice on a topic that has not been mentioned in the TIH and it's very relevant in the hiring process. 

This is the structure of the new sections added:

1. Crafting the Perfect Cover Letter
2. Common Pitfalls to Avoid on your Cover Letter
3. Practical Tips
4. Cover Letter Example and Evaluation

#### Testing Steps:

Tested the changes locally using `yarn dev:website`. Everything looks fine. Screenshot attached of the `live` version and my `localhost:3000` 👇 

image

#### Additional Information:
As we're reviewing content and not code, I'm providing a link to a Google Docs. Feel free to make any comments/suggestions/edits in the document and I'll propagate the changes to the `resume.md` and re-raise the PR.

[[Link to Google Docs](https://docs.google.com/document/d/1Q2-28b1crl_OOQctBwH09TNbLngjKKnCyDEBcRj-j7g/edit#heading=h.y6i2tg9koect)]

_Signed-off-by: [Francisco Vigo] <[fjv222@gmail.com]>_

### CODE REVIEW

**Comment by figo-rabo** [comment]
Forgot to mention. I'm a very commercial Lead Data Scientist so my "Cover Letter" template might be a little bit off for SWE. If we agree on the structure, I can try to find a good template more crafted for Software Engineers.

File: apps/website/contents/resume.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ Having read tons of software engineering resumes as a FAANG interviewer, even so
 Thankfully, there are specific steps and requirements you can fulfill to write a good software engineer resume. From your resume structure, content, to free tools you can use to test your resume, I have collated a very concise summary of the best practices to prepare your resume for a FAANG software engineering job application:
 
 1. Set up an ATS-friendly resume template
-1. Fill up your template with well-framed content in a meaningful order
-1. Optimize your resume with prioritization and keywords
-1. Test out resume using free tools
+2. Fill up your template with well-framed content in a meaningful order
```
Comment (yangshun) [inline apps/website/contents/resume.md:31]: Please revert the changes to the sections to the PR. Using `1.` for all the bullet points is intended, so that we can insert new bullet points anywhere and the rendered numbers are still in order.

File: apps/website/contents/resume.md [hunk 2]
```diff
@@ -77,15 +77,15 @@ ATS readers need to identify and parse standard types of information from your r
...
-| Section | Heading Name |
-| --- | --- |
-| Professional summary | (Use resume headline as section title, for e.g. "Senior Software Engineer at Google with over 5 years of experience leading teams") |
-| Contact information | "Contact Information" |
-| Skills - programming languages, frameworks, etc. | "Skills" |
-| Experience | "Work Experience" |
-| Education (Note: if you are still in school or have less than 3 years of experience, you may put Education first) | "Education" |
-| Projects | "Projects" |
-| Other optional sections - e.g. Certifications, Awards, etc | "Awards and Accolades" / "Certifications" / "Awards, Accolades and Certifications" |
+| Section                                                                                                           | Heading Name                                                                                                                        |
```
Comment (yangshun) [inline apps/website/contents/resume.md:80]: Revert these formatting changes.

File: apps/website/contents/resume.md [hunk 3]
```diff
@@ -343,6 +343,66 @@ Simply copy the content from your resume and paste it into a plain text document
 - Characters are displaying incorrectly in plain text
 - Sections are disorganized
 
+## Crafting the Perfect Cover Letter
+
+Your cover letter serves as a handshake, an introduction to the professional persona encapsulated in your resume. It's an avenue to express your genuine interest in a position or a particular company, portraying the value you can add to the organization. Unlike the formal tone of a resume, a cover letter allows for a unique, creative expression of your personal brand, ensuring it **complements rather than replicates** your CV.
```
Comment (yangshun) [inline apps/website/contents/resume.md:348]: Can we be consistent and use "Resume" instead of "CV" throughout?

File: apps/website/contents/resume.md [hunk 4]
```diff
@@ -343,6 +343,66 @@ Simply copy the content from your resume and paste it into a plain text document
 - Characters are displaying incorrectly in plain text
 - Sections are disorganized
 
+## Crafting the Perfect Cover Letter
```
Comment (yangshun) [inline apps/website/contents/resume.md:346]: Use "Sentence case" for all the headings please

**Review by yangshun — CHANGES_REQUESTED** [review]
Thanks so much! The content looks good, but there are some formatting issues which should be addressed before merging.

File: apps/website/contents/resume.md (hunk 1 above)
Comment (figo-rabo) [reply apps/website/contents/resume.md]: Fixed ✅

File: apps/website/contents/resume.md (hunk 2 above)
Comment (figo-rabo) [reply apps/website/contents/resume.md]: ✅ Fixed. Apologies for this. It took my 30' to figure out what was going on. I had a formatting extension installed that was auto-forcing the changes.

File: apps/website/contents/resume.md (hunk 4 above)
Comment (figo-rabo) [reply apps/website/contents/resume.md]: ✅ Fixed. Updated all titles to be consistent with "Sentence Case".

File: apps/website/contents/resume.md (hunk 3 above)
Comment (figo-rabo) [reply apps/website/contents/resume.md]: ✅ Fixed. Changed all the instances from "CV" to "resume".

**Comment by figo-rabo** [comment]
Hey @yangshun , thanks for the review. I've now resolved all the comments 👍 .

_Changes made before the merge_

1. Reverted bullet points to `1.`
2. Reverted mardown-table formatting to original. 
3. Changed all the instances of `CV` to `resume` for consistency.
4. Updated all the titles to "Sentence Case".
